app/settings.py

```diff
@@ -48,7 +48,8 @@ def read_list(value: str) -> list[str]:
 OSU_API_KEY = os.environ["OSU_API_KEY"]
 
 DOMAIN = os.environ["DOMAIN"]
-MIRROR_URL = os.environ["MIRROR_URL"]
+MIRROR_SEARCH_ENDPOINT = os.environ["MIRROR_SEARCH_ENDPOINT"]
```

Comment:
This is a breaking change. @cmyui how do we handle migrations for stuff like this?

Reply:
Actually true haven't thought about that

Reply:
let him cook